### PR TITLE
New module: Pychopper

### PR DESCRIPTION
**Pull request by caspargross**
- [x] This comment contains a description of changes (with reason)
 - [x] `CHANGELOG.md` has been updated
 - [X] There is example tool output for tools in the pull request https://github.com/ewels/MultiQC_TestData/pull/160
 - [x] Code is tested and works locally (including with `--lint` flag)
 - [x] `docs/README.md` is updated with link to below
 - [x] `docs/modulename.md` is created
 - [x] Everything that can be represented with a plot instead of a table is a plot
 - [x] Report sections have a description and help text (with `self.add_section`)
 - [x] There aren't any huge tables with > 6 columns (explain reasoning if so)
 - [x] Each table column has a different colour scale to its neighbour, which relates to the data (eg. if high numbers are bad, they're red)/
 - [x] Module does not do any significant computational work

### CODE REVIEW

**Comment by caspargross** [comment]
I dont kow why the checks keep failing but to me it looks like the issue is in another module:

```
[ERROR  ]          preseq : Preseq: mixed 'TOTAL_READS' and 'TOTAL_BASES' reports
[WARNING] gene_body_coverage : Empty geneBodyCoverage file found: empty_failed.geneBodyCoverage.txt
[WARNING] junction_saturation : Junction saturation data all zeroes, skipping: 'all_zeroes'
[WARNING]    fastq_screen : Couldn't find number of reads with no hits for 'LIB014528_GEN00035295_TGACCA_L001_R2_screen'
[WARNING]    fastq_screen : Couldn't find number of reads with no hits for 'LIB014528_GEN00035295_TGACCA_L001_R1_screen'
[WARNING]         afterqc : Could not parse AfterQC JSON: 'broken.fq.gz.json'
[WARNING]          fastqc : Couldn't read 'fake_report_fastqc.zip' - Bad zip file
```

**Comment by ewels** [comment]
Ah sorry, that CI test is new and it doesn't give a very informative error message currently. Those error and warning logs are expected and not the problem. What it's testing is that your new module properly raises a `UserWarning` when no files are found. Without this, MultiQC will create a report even if given an empty directory. That's what this test looks for - it runs on an empty directory and checks that no report is generated.

See https://multiqc.info/docs/#no-files-found

**Comment by caspargross** [comment]
Thanks Phil! 
I missed that part of the documentation.

**Comment by ewels** [comment]
No problem, you're not the first! It's also really tricky to spot when reviewing, which is why I finally added that CI test the other day 😄

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
 * [**HOPS**](https://www.github.com/rhubler/HOPS)
   * Post-alignment ancient DNA analysis tool for MALT
 
+* [**Pychopper**](https://github.com/nanoporetech/pychopper)
```
Comment (ewels) [inline CHANGELOG.md:12]:
```suggestion
* [**Pychopper**](https://github.com/nanoporetech/pychopper)
```

File: multiqc/modules/pychopper/pychopper.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            'xDecimals': False,
+            'ymin': 0
+        }
+
+        data_orientation = {}
+        for sample in self.pychopper_data.keys():
+            data_orientation[sample] = {}
+            data_orientation[sample]=self.pychopper_data[sample]['Strand']
+
+        cats = ['+', '-']
```
Comment (ewels) [inline multiqc/modules/pychopper/pychopper.py:143]: I think that this is a _little_ too minimalistic, so worth adding the word `strand` to make the categories a little clearer.

```suggestion
        cats = ['+ strand', '- strand']
```

**Review by ewels — APPROVED** [review]
Code looks great and I it worked really nicely with the example data - a clean run! Just a couple of very minor suggested changes that I'll apply myself now if that's ok.

Many thanks for the contribution 🚀

**Comment by ewels** [comment]
Ah I can't apply the suggestions - I guess because the PR is coming from your org and that means the permissions are a little more locked down than normal so I can't push to your PR.

I'll apply as part of a manual merge 👍🏻 

Apologies also for the long time it's taken me to get to your PR.

**Pull request merged**